test/test_models.py

```diff
@@ -38,65 +38,72 @@ def get_available_video_models():
 # models that are in torch hub, as well as r3d_18. we tried testing all models
 # but the test was too slow. not included are detection models, because
 # they are not yet supported in JIT.
-script_test_models = [
-    "deeplabv3_resnet101",
-    "mobilenet_v2",
-    "resnext50_32x4d",
-    "fcn_resnet101",
-    "googlenet",
-    "densenet121",
-    "resnet18",
-    "alexnet",
-    "shufflenet_v2_x1_0",
-    "squeezenet1_0",
-    "vgg11",
-    "inception_v3",
-    'r3d_18',
+# If 'unwrapper' is provided it will be called with the script model outputs
+# before they are compared to the eager model outputs. This is useful if the
+# model outputs are different between TorchScript / Eager mode
+script_test_models = {
+    "deeplabv3_resnet101": {},
+    "mobilenet_v2": {},
+    "resnext50_32x4d": {},
+    "fcn_resnet101": {},
+    "googlenet": {
+        "unwrapper": lambda x: x.logits
+    },
+    "densenet121": {},
+    "resnet18": {},
+    "alexnet": {},
+    "shufflenet_v2_x1_0": {},
+    "squeezenet1_0": {},
+    "vgg11": {},
+    "inception_v3": {},
+    'r3d_18': {},
+}
+
+
+# These models don't work with checkModule, this list should be deleted as soon
+# as possible
+SCRIPT_MODELS_TO_FIX = [
+    'test_inception_v3',
+    'test_fcn_resnet101',
+    'test_deeplabv3_resnet101',
```

Comment:
I went to https://github.com/pytorch/vision/pull/1352 and the model results stayed the same before and after. Also `IntermediateLayerGetter` is used in other models which correctly return the eager and script result. So it's unclear why these two models are failing. 

When i ran one in script I had to cntrl-C because it was taking so long. I would guess it's a bug somewhere in the JIT runtime.

Reply:
These tests do add a of time to the test exec time (@suo's module changes should help this), this PR uses `PYTORCH_TEST_WITH_SLOW` and `@slowTest` (on by default in the CI) to get around that.